### ISSUE TITLE
Do not wait too long for couchdb/indexer/controller

### ISSUE THREAD

**Issue by nicofrand** [cozy/cozy-debian#13]
If couchdb or cozy-indexer or cozy-controller won't start at all for any reason, the installation process keeps running indefinitely due to :

https://github.com/cozy/cozy-debian/blob/master/debian/postinst#L44
https://github.com/cozy/cozy-debian/blob/master/debian/postinst#L72
https://github.com/cozy/cozy-debian/blob/master/debian/postinst#L86

The real issue is that the installation cannot be aborted by pressing `CTRL+V`.

Maybe the loops should stop if 50 iterations already occurred ? The user could still try to install cozy again by re-running `apt-get install cozy` ?

**Comment by nledez** [comment]
Fixed with 83291f0fd70e6f503ae84ef5387c5fe0fd56a1a6.
@nicofrand When the package is released, can you try it and close the issue if you thinks it's done.

Side effect: the Ctrl-C work now :)

**Comment by nicofrand** [comment]
I will.
The installation is blocked when waiting for the controlller if the cozy-controller.conf is incorrect. To check if the patch works, simply removing the file and making a "touch" should be enough:

``` console
rm /etc/supervisor/conf.d/cozy-indexer.conf
touch /etc/supervisor/conf.d/cozy-indexer.conf
```
